app/models/article.js

```diff
@@ -0,0 +1,12 @@
+import DS from 'ember-data';
+
+let Article = DS.Model.extend({
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).